### ISSUE TITLE
Promise polyfill throws "Can't find variable: _enqueueMicrotask" error

### ISSUE THREAD

**Issue by jr-codes** [polyfillpolyfill/polyfill-service#422]
I get this error when I load the Promise polyfill in IE10 or iOS7 Safari:

```
ReferenceError: Can't find variable: _enqueueMicrotask
```

Here's the URL I used: http://cdn.polyfill.io/v1/polyfill.min.js?features=default,Promise&gated=1

**Comment by triblondon** [comment]
Hi @zarjay, could you do the following to help us diagnose this?
1. Change the URL to drop the `.min` and confirm whether the problem is still reproducible
2. If it is, copy the comment at the top of the resulting HTTP response.  In IE10 it _should_ look like this:

```
/* For detailed credits and licence information see http://github.com/financial-times/polyfill-service.
 * 
 * UA detected: ie/10.0.0
 * Features requested: Promise,default
 * Library version: latest
 * 
 * - _enqueueMicrotask, License: CC0  (required by "Promise")
 * - Promise, License: MIT 
 * - Event, License: CC0  (required by "default", "CustomEvent")
 * - CustomEvent, License: CC0  (required by "default")
 * - DOMTokenList-toggle, License: CC0  (required by "default")
 * - Element.prototype.matches-ms, License: CC0  (required by "default", "Element.prototype.closest")
 * - Element.prototype.closest, License: CC0  (required by "default")
 * - Object.assign, License: CC0  (required by "default")
 * - String.prototype.includes, License: CC0  (required by "default") */
```

**Comment by jr-codes** [comment]
No longer able to reproduce it in IE10, but I'm still able to reproduce it in iOS7. Here's what I see in IE10:

```
/* For detailed credits and licence information see http://github.com/financial-times/polyfill-service.
 * 
 * UA detected: ie/10.0.0
 * Features requested: Promise,default
 * Library version: latest
 * 
 * - _enqueueMicrotask, License: CC0  (required by "Promise")
 * - Promise, License: MIT 
 * - Event, License: CC0  (required by "default", "CustomEvent")
 * - CustomEvent, License: CC0  (required by "default")
 * - DOMTokenList-toggle, License: CC0  (required by "default")
 * - Element.prototype.matches-ms, License: CC0  (required by "default", "Element.prototype.closest")
 * - Element.prototype.closest, License: CC0  (required by "default")
 * - Object.assign, License: CC0  (required by "default")
 * - String.prototype.includes, License: CC0  (required by "default") */
```

And here's what I see in iOS7 Safari:

```
/* For detailed credits and licence information see http://github.com/financial-times/polyfill-service.
 * 
 * UA detected: ios_saf/7.0.0
 * Features requested: Promise,default
 * Library version: latest
 * 
 * - Promise, License: MIT 
 * - Element.prototype.matches-webkit, License: CC0  (required by "default", "Element.prototype.closest")
 * - Element.prototype.closest, License: CC0  (required by "default")
 * - Object.assign, License: CC0  (required by "default")
 * - String.prototype.includes, License: CC0  (required by "default") */
```

Here's a [simple test in CodePen](http://codepen.io/jr/full/WvzwLb/), which returns false in my iOS7.1 simulator.

**Comment by triblondon** [comment]
Ah, easy fix - it looks like one of our 'private polyfills' (essentially functions that we share between polyfills but which do not emulate native features) did not list every browser in its config correctly.  Private polyfills must list every browser because by definition no browser has the feature.

Will be released in 1.5.
